README.md

```diff
@@ -30,6 +30,14 @@ environment variables override those:
 
 `NIPIO_SOA_NS`: SOA name server.
 
+`NIPIO_SOA_REFRESH`: SOA refresh.
+
+`NIPIO_SOA_RETRY`: SOA retry.
+
+`NIPIO_SOA_EXPIRY`: SOA expiry.
+
+`NIPIO_SOA_MINIMUM`: SOA minimum ttl.
```

Comment:
Uppercase TTL since it is an acronym. Expand TTL into "SOA minimum time-to-live (TTL)".

Reply:
Perhaps `NIPIO_SOA_MINIMUM_TTL` would describe it better?